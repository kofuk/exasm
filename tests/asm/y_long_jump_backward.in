@beg nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
nop
j @beg
nop
