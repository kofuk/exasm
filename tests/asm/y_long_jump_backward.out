@00 00000000 00000000 // nop
@02 00000000 00000000 // nop
@04 00000000 00000000 // nop
@06 11000000 00000110 // j 0x06
@08 00000000 00000000 // nop
@0a 11000000 11110100 // j -0x0C
@0c 00000000 00000000 // nop
@0e 00000000 00000000 // nop
@10 00000000 00000000 // nop
@12 00000000 00000000 // nop
@14 00000000 00000000 // nop
@16 00000000 00000000 // nop
@18 00000000 00000000 // nop
@1a 00000000 00000000 // nop
@1c 00000000 00000000 // nop
@1e 00000000 00000000 // nop
@20 00000000 00000000 // nop
@22 00000000 00000000 // nop
@24 00000000 00000000 // nop
@26 00000000 00000000 // nop
@28 00000000 00000000 // nop
@2a 00000000 00000000 // nop
@2c 00000000 00000000 // nop
@2e 00000000 00000000 // nop
@30 00000000 00000000 // nop
@32 00000000 00000000 // nop
@34 00000000 00000000 // nop
@36 00000000 00000000 // nop
@38 00000000 00000000 // nop
@3a 00000000 00000000 // nop
@3c 00000000 00000000 // nop
@3e 00000000 00000000 // nop
@40 00000000 00000000 // nop
@42 00000000 00000000 // nop
@44 00000000 00000000 // nop
@46 00000000 00000000 // nop
@48 00000000 00000000 // nop
@4a 00000000 00000000 // nop
@4c 00000000 00000000 // nop
@4e 00000000 00000000 // nop
@50 00000000 00000000 // nop
@52 00000000 00000000 // nop
@54 00000000 00000000 // nop
@56 00000000 00000000 // nop
@58 00000000 00000000 // nop
@5a 00000000 00000000 // nop
@5c 00000000 00000000 // nop
@5e 00000000 00000000 // nop
@60 00000000 00000000 // nop
@62 00000000 00000000 // nop
@64 00000000 00000000 // nop
@66 00000000 00000000 // nop
@68 00000000 00000000 // nop
@6a 00000000 00000000 // nop
@6c 00000000 00000000 // nop
@6e 00000000 00000000 // nop
@70 00000000 00000000 // nop
@72 00000000 00000000 // nop
@74 00000000 00000000 // nop
@76 00000000 00000000 // nop
@78 00000000 00000000 // nop
@7a 00000000 00000000 // nop
@7c 00000000 00000000 // nop
@7e 00000000 00000000 // nop
@80 00000000 00000000 // nop
@82 00000000 00000000 // nop
@84 00000000 00000000 // nop
@86 00000000 00000000 // nop
@88 11000000 10000000 // j -0x80
@8a 00000000 00000000 // nop
